toc/ROLES.md

```diff
@@ -75,7 +75,7 @@ establish sub-working groups. Working groups delegate change approval to Approve
     <td>
         <p>Can get PRs accepted</p>
     </td>
-    <td>GitHub Organization</td>
+    <td>Per GitHub Organization</td>
```

Comment:
"Per Github Org" or for "all CFF Github Orgs"?
Is it necessary to sign the CLA per GitHub Org?